pipelines/broad/rna_seq/BroadInternalRNAWithUMIsPipeline.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+import "RNAWithUMIsPipeline.wdl" as RNAWithUMIsPipeline
+import "../../../tasks/broad/Utilities.wdl" as utils
+
+
+workflow BroadInternalRNAWithUMIsPipeline {
+
+  String pipeline_version = "0.1.0"
+
+  input {
+    # input needs to be either "hg19" or "b38"
```

Comment:
Can we use hg19 or hg38 as the input? It would be more consistent with the rest of our pipelines and files

Reply:
sounds good to me